Makefile.menhir

```diff
@@ -47,8 +47,9 @@ promote-menhir: parsing/parser.mly
 .PHONY: test-menhir
 test-menhir: parsing/parser.mly
 	$(MENHIR) $(MENHIRFLAGS) parsing/parser.mly
-	sed -i "s/MenhirLib/CamlinternalMenhirLib/g" \
-	  $(addprefix parsing/parser.,ml mli)
+	sed -e "s/MenhirLib/CamlinternalMenhirLib/g" -i.bak \
+	  $(addprefix parsing/parser.,ml mli) \
+	  && rm parsing/parser.*.bak
```

Comment:
I shouldn't have used `-i` in the first place, but I have no idea what `-i.bak` is and I don't want to hear about it. Would you be patient enough to just use `cat | sed > foo.tmp; mv foo.tmp foo`?